packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -69,7 +70,13 @@ export const ArticlePostCard = forwardRef(function PostCard(
       ref={ref}
       flagProps={{ pinnedAt, trending }}
     >
-      <CardButton title={post.title} onClick={onPostCardClick} />
+      <CardButton
```

Comment:
Why don't we just not render this when it's `isFeedPreview`?
(not really sure what it adds now)

Reply:
Yep, good point.